captum/attr/_models/base.py

```diff
@@ -5,10 +5,10 @@
 import warnings
 
 from functools import reduce
-from torch.nn import Embedding
+from torch.nn import Module
 
 
-class InterpretableEmbeddingBase(Embedding):
+class InterpretableEmbeddingBase(Module):
```

Comment:
Should this now be `InterpretableModuleBase` or is it still only Embedding even though you are inheriting from Module?

Reply:
Actually, what's the specific motivation for inheriting differently? Seems like you'd want to still just have Embedding per the code. If you still need this please add a good comment about why.

Reply:
Good point! In a general case Embedding doesn't have to extend `nn.Embedding` but an `nn.Module`. An example of it is `BertEmbedding`:https://github.com/huggingface/transformers/blob/master/transformers/modeling_bert.py#L151